test/functionalTest/cases/0000_content_related_headers/accept_fail.test

```diff
@@ -25,12 +25,32 @@ Accept header fail
 
 --SHELL-INIT--
 dbInit CB
-brokerStart CB
+brokerStart CB 38,39
```

Comment:
Yes.
This test failed (that's how I discovered the bug in connectionTreat.
In order toi understandthe test I had to 'transform' it to something readable